build/yamls/flow-aggregator.yml

```diff
@@ -203,12 +203,47 @@ data:
 
       # TLS min version from: VersionTLS10, VersionTLS11, VersionTLS12, VersionTLS13.
       #tlsMinVersion:
+
+    # clickHouse contains ClickHouse related configuration options.
+    #clickHouse:
+      # Enable is the switch of enabling exporting flow records to ClickHouse.
+      #enable: false
+
+      # Database is the name of database where Antrea "flows" table is created. Defaults to "default".
+      #database: "default"
+
+      # DatabaseURL is the url to the database. TCP protocol is required.
+      # Defaults to "tcp://clickhouse-clickhouse.flow-visibility.svc:9000".
+      #databaseURL: "tcp://clickhouse-clickhouse.flow-visibility.svc:9000"
+
+      # Debug enables debug level log from clickhouse sql driver.
+      #debug: false
+
+      # Compress enables lz4 compression when committing flow records. Defaults to true.
+      #compress: true
+
+      # CommitInterval is the periodical interval between batch commit of flow records to DB.
+      # Defaults to "8s". Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
+      # Per ClickHouse, this interval should not be shorter than 1s for best performance.
+      #commitInterval: "8s"
```

Comment:
If we allow flow aggregator to use intervals <1sec, could we end up hanging the flow aggregator or the clickhouse DB? Not something that must be address in this PR, but I think we need to consider a lower bound on the commitInterval.

Reply:
sure. will add additional check alongside the config format change.

Reply:
added a lower bound check for 1s. in reality intervals shorter than 1s alone won't break CH given low flow record ingestion rate. We may want to relax the bound by a bit but I don't see a sensible difference for end user.